pkg/trisa/keys/signature/signature.go

```diff
@@ -0,0 +1,134 @@
+/*
+Package signature provides a mechanism for computing public key signatures, which are
+used to help identify public keys used in sealing TRISA envelopes and select the
+matching private key pair when a secure envelope is received.
+
```

Comment:
Worth it to add a recommendation in line with the white paper w/r/t sealing key usage? E.g.

TRISA strongly recommends that sealing/unsealing keys be distinct from identity certificates for improved security and to support differing retirement criteria. Some organizations may choose to use unique sealing/unsealing keys with each unique counterparty.